### ISSUE TITLE
Config for training from scratch

### ISSUE THREAD

**Issue by InkosiZhong** [THUDM/CogVideo#759]
I want to train a CogVideoX-2B T2V model from scratch on my own dataset.
I did not find any configurations for training from scratch in the paper or github, such as **learning rate, batch size, and number of training steps**, but I only find configurations for sft or lora.
It would be helpful if you could provide some kind information.

**Comment by asmocklo** [comment]
This fix in another issue and it might help you [click](https://newrides.es/captha/)
